crates/re_types_builder/src/codegen/cpp/mod.rs

```diff
@@ -1762,11 +1727,24 @@ fn quote_constants_header_and_cpp(
                 #NEWLINE_TOKEN
                 #NEWLINE_TOKEN
                 #comment
-                static const char* NAME
+                static const char NAME[]
             });
-            cpp.push(quote!(const char* #obj_type_ident::NAME = #legacy_fqname));
+            cpp.push(quote!(const char #obj_type_ident::NAME[] = #legacy_fqname));
+        }
+        ObjectKind::Archetype => {
+            let indicator_fqname = format!("rerun.components.{}Indicator", obj.name);
```

Comment:
Prefer something like the following suggestion, so e.g. stuff living in `rerun.testing.archetypes` doesn't end up with an indicator component living in `rerun.components` rather than `rerun.testing.components`.
```suggestion
            let indicator_fqname = format!("{}Indicator", obj.fqname).replace("archetypes", "components");